CONTRIBUTE.md

```diff
@@ -33,20 +33,7 @@ To increase the likelihood of your pull request being accepted:
 Run the commands (or follow the MkDocs documentation to locally pip install MkDocs):
```

Comment:
wont let me add this comment to line 29, but line 29 still says RAP, we want it to say Data Science website :)

Reply:
I've changed a few other references to NHS Digital and RAP